guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -91,7 +95,13 @@ Ensure {SmartProxy} has access to `{RepoRHEL8ServerSatelliteMaintenanceProductVe
 +
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-maintain} self-upgrade
```

Comment:
This line is now dropped. I don't think this is valid.

Reply:
Yeah, the `self-upgrade` step should still be here.

Reply:
Restored :+1: